{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2017-04-04T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Django 1.10 before 1.10.7, 1.9 before 1.9.13, and 1.8 before 1.8.18 relies on user input in some cases to redirect the user to an \"on success\" URL. The security check for these redirects (namely ``django.utils.http.is_safe_url()``) considered some numeric URLs \"safe\" when they shouldn't be, aka an open redirect vulnerability. Also, if a developer relies on ``is_safe_url()`` to provide safe redirect targets and puts such a URL into a link, they could suffer from an XSS attack."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2018-10-17T09:57:01",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "1038177",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_SECTRACK"
                    ],
                    "url": "http://www.securitytracker.com/id/1038177"
                },
                {
                    "name": "RHSA-2017:1596",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2017:1596"
                },
                {
                    "name": "97406",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_BID"
                    ],
                    "url": "http://www.securityfocus.com/bid/97406"
                },
                {
                    "name": "RHSA-2017:3093",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2017:3093"
                },
                {
                    "name": "DSA-3835",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_DEBIAN"
                    ],
                    "url": "http://www.debian.org/security/2017/dsa-3835"
                },
                {
                    "name": "RHSA-2017:1445",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2017:1445"
                },
                {
                    "name": "RHSA-2017:1451",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2017:1451"
                },
                {
                    "name": "RHSA-2018:2927",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2018:2927"
                },
                {
                    "name": "RHSA-2017:1470",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2017:1470"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.djangoproject.com/weblog/2017/apr/04/security-releases/"
                },
                {
                    "name": "RHSA-2017:1462",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2017:1462"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2017-7233",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "Django 1.10 before 1.10.7, 1.9 before 1.9.13, and 1.8 before 1.8.18 relies on user input in some cases to redirect the user to an \"on success\" URL. The security check for these redirects (namely ``django.utils.http.is_safe_url()``) considered some numeric URLs \"safe\" when they shouldn't be, aka an open redirect vulnerability. Also, if a developer relies on ``is_safe_url()`` to provide safe redirect targets and puts such a URL into a link, they could suffer from an XSS attack."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "1038177",
                            "refsource": "SECTRACK",
                            "url": "http://www.securitytracker.com/id/1038177"
                        },
                        {
                            "name": "RHSA-2017:1596",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2017:1596"
                        },
                        {
                            "name": "97406",
                            "refsource": "BID",
                            "url": "http://www.securityfocus.com/bid/97406"
                        },
                        {
                            "name": "RHSA-2017:3093",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2017:3093"
                        },
                        {
                            "name": "DSA-3835",
                            "refsource": "DEBIAN",
                            "url": "http://www.debian.org/security/2017/dsa-3835"
                        },
                        {
                            "name": "RHSA-2017:1445",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2017:1445"
                        },
                        {
                            "name": "RHSA-2017:1451",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2017:1451"
                        },
                        {
                            "name": "RHSA-2018:2927",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2018:2927"
                        },
                        {
                            "name": "RHSA-2017:1470",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2017:1470"
                        },
                        {
                            "name": "https://www.djangoproject.com/weblog/2017/apr/04/security-releases/",
                            "refsource": "CONFIRM",
                            "url": "https://www.djangoproject.com/weblog/2017/apr/04/security-releases/"
                        },
                        {
                            "name": "RHSA-2017:1462",
                            "refsource": "REDHAT",
                            "url": "https://access.redhat.com/errata/RHSA-2017:1462"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2017-7233",
        "datePublished": "2017-04-04T17:00:00",
        "dateReserved": "2017-03-22T00:00:00",
        "dateUpdated": "2018-10-17T09:57:01",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}